notifications_utils/version_tools/requirements_for_test_common.in

```diff
@@ -7,5 +7,4 @@ pytest-testmon==2.1.1
 requests-mock==1.12.1
 freezegun==1.5.1
 
-black==24.10.0  # Also update `.pre-commit-config.yaml` if this changes
 ruff==0.8.2  # Also update `.pre-commit-config.yaml` if this changes
```

Comment:
while we're changing our formatter, is it worth updating to ruff 0.9.x? (which changes formatter rules slightly, including "Improved compatibility with Black’s style guide")

https://astral.sh/blog/ruff-v0.9.0

Reply:
> Preserving the source formatting for these cases means that the [longstanding formatter incompatibility](https://github.com/astral-sh/ruff/issues/8272) with our [ISC001 rule](https://docs.astral.sh/ruff/rules/single-line-implicit-string-concatenation/) is now resolved. Users will no longer see a warning if they have both the formatter and this lint rule enabled in their Ruff configuration.

Reply:
Good idea, have made the bump in 122d4fd and rebased